cylc/flow/config.py

```diff
@@ -2278,6 +2284,25 @@ def load_graph(self):
         for tdef in self.taskdefs.values():
             tdef.tweak_outputs()
 
+    def check_outputs(self, terminals: Iterable[str]) -> None:
```

Comment:
If this is only checking "terminals", maybe call the method `check_terminal_outputs`.

Would be good to define "terminal output" in the docstring as it's a very niche term.

Reply:
```suggestion
    def check_terminal_outputs(self, terminals: Iterable[str]) -> None:
```